scripts/base/frameworks/telemetry/main.zeek

```diff
@@ -0,0 +1,505 @@
+##! Module providing helpers to collect telemetry. This wraps the
+##! somewhat lower-level telemetry.bif functions.
+
+module Telemetry;
+
+export {
+	type labels_table: table[string] of string;
+	type labels_vector: vector of string;
+
+	## Default bounds/buckets for histograms if no bounds are explicitly provided.
+	const default_histogram_bounds = vector(0.01, 0.1, 1.0, 10.0, 30.0, 60.0) &redef;
```

Comment:
Wondering if a default is useful given that semantics of bounds are very metric-specific?

Reply:
Removed now. Other libraries have defaults, but also agree may be too metric specific.